### PR TITLE
scripts : update scripts directory

### PR DESCRIPTION
**Pull request by RuffaloLavoisier**
*  Creating a directory.
*  Delete Remaining Files(fetch file).
*  Set a new file path to a folder for screenshots(go to `screenshot/`).

Keep your messy storage clean !

### CODE REVIEW

File: createScreenshot.sh [hunk 1]
```diff
@@ -6,6 +6,6 @@ then
     exit
 fi
 
-bash fetchScreen.sh $1 $2.png
+bash screenshotScript/fetchScreen.sh $1 $2.png
```
Comment (simonmicro) [inline createScreenshot.sh:9]: Maybe call it `scripts`, then we could also migrate our Python stuff there.

**Review by simonmicro — CHANGES_REQUESTED** [review]
Just minor stuff, overall idea is OK

File: Readme.md [hunk 2]
```diff
@@ -63,6 +63,12 @@ http://<IP_OF_WATCH>/api/screenserver
 
 The `fetchScreen.sh` downloads the raw image buffer from the running screen server, and converts the image to png. The `composeScreen.sh` creates the image with a surrounding smartwatch (light edition).
 
+#### Quick-Fast (recommend)
```
[the remark this answers is not in the document]
Comment (RuffaloLavoisier) [reply Readme.md:66]: Thanks 🤣 🤣 🤣 🤣 🤣 🤣 🤣

File: createScreenshot.sh (hunk 1 above)
Comment (RuffaloLavoisier) [reply createScreenshot.sh:9]: Hmm.. how about `Tools`?

File: Readme.md [hunk 3]
```diff
@@ -63,6 +63,12 @@ http://<IP_OF_WATCH>/api/screenserver
...
+```
+$ cd open-smartwatch-os/
+$ ./createScreenshot.sh <IP_OF_WATCH> <SCREENSHOT>
```
[the remark this answers is not in the document]
Comment (RuffaloLavoisier) [reply Readme.md:69]: Where are you talking about?

[the remark this answers is not in the document]
Comment (RuffaloLavoisier) [reply Readme.md:69]: I understood what it meant.

[the remark this answers is not in the document]
Comment (RuffaloLavoisier) [reply Readme.md]: Maybe .. you mean (94b7148c)?

File: createScreenshot.sh (hunk 1 above)
Comment (RuffaloLavoisier) [reply createScreenshot.sh:9]: I think we should move all the script tools(`*.py`,`*.sh`...etc).
```
Open-smartwatch/
└── scripts/
        ├── Python/
        └── Shell/
```

Comment (uvwxy) [reply createScreenshot.sh:9]: `scripts` get's my vote 👍

Comment (RuffaloLavoisier) [reply createScreenshot.sh:9]: Fair

**Comment by RuffaloLavoisier** [comment]
```
../scripts/
├── build_install
│   ├── install.py
│   ├── prebuild_cppflags.py
│   ├── prebuild_info.py
│   ├── prebuild_lua.py
│   └── prebuild_www.py
├── screen_capture
│   ├── composeScreen.sh
│   ├── createScreenshot.sh
│   ├── fetchScreen.sh
│   ├── screenshot-overlay-mask.png
│   └── screenshot-overlay.png
└── update_sync
    └── update.sh
 ```
 
 Now..directory

**Comment by RuffaloLavoisier** [comment]
We need to rename the directory of the various scripts as appropriate.

**Pull request merged**